_data/simple-icons.json

```diff
@@ -5538,6 +5538,11 @@
             "hex": "000000",
             "source": "https://github.com/WordPress/gutenberg/blob/7829913ae117dfb561d14c600eea7b281afd6556/docs/final-g-wapuu-black.svg"
         },
+        {
+            "title": "H&M",
+            "hex": "E50010",
+            "source": "https://www.hm.com/entrance.ahtml"
```

Comment:
Since the logo in this page is a PNG, I would use the lin from a selected region (probably the [US](https://www2.hm.com/en_us/index.html) one) where a SVG is available.

Reply:
Yeah I meant to replace it with either the UK link or the 'global' site.